assembl/static2/js/app/components/administration/discussion/languageAdmin.jsx

```diff
@@ -6,11 +6,31 @@ import { Checkbox } from 'react-bootstrap';
 
 import SectionTitle from '../sectionTitle';
 import withLoadingIndicator from '../../common/withLoadingIndicator';
+import { addLanguagePreferences, removeLanguagePreference } from '../../../actions/adminActions';
 import getAllPreferenceLanguage from '../../../graphql/AllLanguagePreferences.graphql';
 
 
 class LanguageSection extends React.Component {
 
+  constructor(props) {
+    super(props);
+    const {i18n, selectedLocale, data } = props;
+
+    const allLangs = {};
+    data.defaultPreferences.languages.forEach((lang) => {
+      allLangs[lang.locale] = {selected: false, name: lang.name};
```

Comment:
ok this solves my previous remark of not using "state" but a more meaningful word like "active", so "selected" be it!